script.js

```diff
@@ -1,4 +1,176 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (
+    input != "scissors" ||
+    input != "paper" ||
+    input != "rock" ||
+    input != "reversed scissors" ||
+    input != "reversed paper" ||
+    input != "reversed rock"
+  ) {
+    myOutputValue =
+      "Please try again and input the following : Scissors, Paper, Rock, Reversed Scissors, Reversed Paper, Reversed Rock";
+  }
+  var randomChoice = generateRandomChoice();
+  var randomChoiceReversed = generateRandomChoiceReversed(); // for reversed game
+  console.log("randomChoice");
+  console.log(randomChoice);
+
+  // draw scenarios
+  if (input == randomChoice) {
+    myOutputValue =
+      "The computer chose " +
+      randomChoice +
+      ". <br>" +
+      "You chose " +
+      input +
+      ". <br><br>" +
+      "It's a draw! <br><br>" +
+      "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! ";
+  }
```

Comment:
can consider using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to make it easier to write this part